File: app/src/main/java/widget/CounterWidget.java
package widget;

import asvid.beercounter.App;
import asvid.beercounter.data.CounterItem;
import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

/**
 * Created by adam on 15.01.17.
 */

public class CounterWidget extends RealmObject {

    @PrimaryKey
    private long id;
    private CounterItem counterItem;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public CounterItem getCounterItem() {
        return counterItem;
    }

    public void setCounterItem(CounterItem counterItem) {
        this.counterItem = counterItem;
    }
}

File: app/src/main/java/widget/CounterWidget.kt
package widget

import asvid.counter.data.CounterItem
import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

/**
 * Created by adam on 15.01.17.
 */

open class CounterWidget : RealmObject() {

    @PrimaryKey
    open var id: Long? = null

    open var counterItem: CounterItem? = null

    override fun toString(): String {
        return "CounterWidget(id=$id, counterItem=$counterItem)"
    }

}
